packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -10778,6 +11102,29 @@ components:
       enum:
         - REST
         - SOAP
+    EServiceDescriptorAttributes:
+      description: |
+        A list of groups of attributes associated with an E-Service descriptor.
+        An E-Service descriptor can have multiple attribute groups for each
+        attribute kind (certified, declared, verified).
+        Groups of attributes are considered in AND (i.e., must be satisfied together),
+        while attributes within the same group are considered in OR.
+        You can model a single attribute as a group with one element.
+      type: array
```

Comment:
to allow future enhancements, request bodies and responses should be `object`s.
In this case it should be an object that contains an array of arrays